src/util/json.cpp

```diff
@@ -133,6 +120,41 @@ void jsont::output_rec(std::ostream &out, unsigned indent) const
   }
 }
 
+/// Basic handling of the printing of a JSON object.
+/// Dispatches to output_rec for most of the hard work.
+/// \param out: The stream that the JSON object is to be
+///   printed to.
+/// \param object: The JSON object.
+/// \param indent: The indentation level.
+void jsont::output_object(
+  std::ostream &out,
+  const objectt &object,
+  unsigned indent)
+{
+  for(objectt::const_iterator o_it=object.begin();
+      o_it!=object.end();
+      o_it++)
+  {
+    if(o_it!=object.begin())
+      out << ',';
+
+    out << '\n';
```

Comment:
Shouldn't the newline also be conditional on `o_it!=object.beginn()`?

Reply:
No, because an object always starts with an opening brace, after which we put a newline (code snippet factored out from lines 69-85 above).